scripts/utility/adminClient.sh

```diff
@@ -15,6 +15,11 @@
 
 set -e
 
+if [[ -z $1 ]]; then
+	echo "Usage: $0 <instance-id>"
+	exit 1
+fi
```

Comment:
These indents look like they're too wide compared to the other script files

Reply:
Fixed. That was copy-pasted from another script, I've fixed the indentation there too.